userguide/system.rst

```diff
@@ -221,7 +221,9 @@ viewed or set using
 :menuselection:`System --> System Dataset`.
 
 .. note:: :ref:`SSH` keys are not stored in the configuration database
-   and must be backed up separately.
+   and must be backed up separately. System host keys are stored in
+   :file:`/usr/local/etc/ssh/ssh_hosts*`. The root user keys are
```

Comment:
Please cherry-pick the changes we did in the other one, with the name fixed to `ssh_host` and not assuming the user knows what the wildcard means.